sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/converters/AddChatParticipantsOptionsConverter.java

```diff
@@ -12,24 +11,24 @@
 
 /**
  * A converter between {@link com.azure.android.communication.chat.implementation.models.AddChatParticipantsOptions} and
- * {@link AddChatParticipantsOptions}.
+ * a collection of {@link ChatParticipant}.
```

Comment:
What about the following?

```suggestion
 * A {@link ChatParticipant} Iterable.
```